src/Files.App/Data/Factories/ShellContextFlyoutHelper.cs

```diff
@@ -180,17 +180,11 @@ async Task InvokeShellMenuItemAsync(ContextMenu contextMenu, object? tag)
 				switch (verb)
 				{
 					case "install" when isFont:
-						{
-							foreach (string path in contextMenu.ItemsPath)
-								await Win32API.InstallFont(path, false);
-						}
+							await Task.WhenAll(contextMenu.ItemsPath.Select(path => Win32API.InstallFont(path, false)));
 						break;
 
 					case "installAllUsers" when isFont:
-						{
-							foreach (string path in contextMenu.ItemsPath)
-								await Win32API.InstallFont(path, true);
-						}
+						await Task.WhenAll(contextMenu.ItemsPath.Select(path => Win32API.InstallFont(path, true)));
```

Comment:
We should use `InstallFontsAsync()` in the first place.

Reply:
👍